src/presentation/ui/page/accounts.templ

```diff
@@ -152,7 +203,8 @@ templ AccountsTable(accounts []entity.Account) {
 		</thead>
 		<tbody>
 			for _, account := range accounts {
-				<tr class="odd:bg-os-300 even:bg-os-500 border-b border-neutral-500 border-opacity-30 text-center">
+				@templ.JSONScript("secureAccessKeys_"+account.Id.String(), account.SecureAccessKeys)
```

Comment:
I know this is hidden, but shouldn't it be inside a <td> so we don't break the <table> parsing? Some browsers may have a tough timing parsing the HTML in this situation I guess.